### ISSUE TITLE
2.02 seems to be broken

### ISSUE THREAD

**Issue by ctbritt** [trioderegion/dnd5e-helpers#115]
Hi, Just updated to 2.02 and running 0.8.5. In the previous version (2.01), module was running fine in computing cover and having automatic action handling. Now, neither work. There's no action menu on the token HUD and computing cover is non-active. 

No other mods activated.

Here's the error I'm getting:
```
Uncaught (in promise) TypeError: WallsLayer.getRayCollisions is not a function
    at dnd5e-helpers.js:2897
    at Array.map (<anonymous>)
    at pointToSquareCover (dnd5e-helpers.js:2895)
    at dnd5e-helpers.js:2828
    at Array.map (<anonymous>)
    at dnd5e-helpers.js:2827
    at Array.map (<anonymous>)
    at Token5e.Token.computeTargetCover (dnd5e-helpers.js:2824)
    at onTargetToken (dnd5e-helpers.js:2564)
    at dnd5e-helpers.js:864
(anonymous) @ dnd5e-helpers.js:2897
pointToSquareCover @ dnd5e-helpers.js:2895
(anonymous) @ dnd5e-helpers.js:2828
(anonymous) @ dnd5e-helpers.js:2827
Token.computeTargetCover @ dnd5e-helpers.js:2824
onTargetToken @ dnd5e-helpers.js:2564
(anonymous) @ dnd5e-helpers.js:864
_call @ foundry.js:194
callAll @ foundry.js:153
_hook @ foundry.js:37189
add @ foundry.js:37168
setTarget @ foundry.js:44987
_onClickLeft @ foundry.js:45373
callback @ foundry.js:35310
_handleClickLeft @ foundry.js:35497
_handleMouseDown @ foundry.js:35477
a.emit @ index.js:181
r.dispatchEvent @ interaction.min.js:8
r.processPointerDown @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.findHit @ interaction.min.js:8
r.processInteractive @ interaction.min.js:8
r.onPointerDown @ interaction.min.js:8
async function (async)
onTargetToken @ dnd5e-helpers.js:2564
(anonymous) @ dnd5e-helpers.js:864
_call @ foundry.js:194
callAll @ foundry.js:153
_hook @ foundry.js:37189
add @ foundry.js:37168
setTarget @ foundry.js:44987
_onClickLeft @ foundry.js:45373
callback @ foundry.js:35310
_handleClickLeft @ foundry.js:35497
_handleMouseDown @ foundry.js:35477
a.emit @ index.js:181
r.dispatchEvent @ interaction.min.js:8
r.processPointerDown @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.recursiveFindHit @ interaction.min.js:8
t.findHit @ interaction.min.js:8
r.processInteractive @ interaction.min.js:8
r.onPointerDown @ interaction.min.js:8
```

**Comment by trioderegion** [comment]
Nothing with collisions has been touched since 2.0, so I'm not sure why its only breaking now.  However, 5e helpers is not expected to be compatible with 0.8.x builds at the moment.

Will note this for 0.8.x tasks.

**Comment by trioderegion** [comment]
Fixed in the upcoming release for 0.8.6
